cirq/sim/sparse_simulator.py

```diff
@@ -365,6 +365,18 @@ def state_vector(self, copy: bool = True):
         return vector.copy() if copy else vector
 
     def set_state_vector(self, state: 'cirq.STATE_VECTOR_LIKE'):
+        """Set the state vector.
+
+        One can pass a valid full state to this method by passing a numpy
+        array. Or, alternatively, one can pass an integer, and then the state
+        will be set lie entirely in the computation basis state for the binary
```

Comment:
"will be set lie" -> "will be set to lie"

Same in the notebook.

Reply:
Done.